modules/data-mover-prerequisites.adoc

```diff
@@ -0,0 +1,11 @@
+// This is a small bit of text that will need to be copy-pasted into the procedure installing the DataMover feature into OADP 1.1.0.
+
+
+.Prerequisite
+
+* VolSync Operator installed using the Operator Lifecycle Manager (OLM). Administrators can link:https://docs.openshift.com/container-platform/4.10/operators/admin/olm-adding-operators-to-cluster.html[install Operators on clusters]. Non-administrators can link:https://docs.openshift.com/container-platform/4.10/operators/user/olm-installing-operators-in-namespace.html[install Operators in namespaces].
+
+[NOTE]
+====
+The VolSync Operator is required only for use with the Technology Preview DataMover. The Operator is not required in other to use OADP GA features.
```

Comment:
@RichardHoch 
"...in other to use..."  What did you want to say here? "in order to use"?

Reply:
Fixed.